examples/pipelines/ccfraud/submit.py

```diff
@@ -279,6 +312,15 @@ def fl_ccfraud_basic():
             # make sure the compute corresponds to the silo
             silo_training_step.compute = silo_config.compute
 
+            # set distribution according to the number of available GPUs (1 in case of only CPU available)
```

Comment:
How about using an additional flag while submitting a pipeline(maybe --distributed)? n you can put the distributed code in a separate block or fn. In that way, you can keep both the existing as well as the distributed code separate. That would also resolve the MLClient issue.

Reply:
Hmm, is it in any way issue that the MLClient is always created?